drivers/oled/oled_driver.h

```diff
@@ -150,6 +150,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #    endif
 #endif
 
+#if defined(__GNUC__)
+#    define PACKED __attribute__((__packed__))
+#else
+#    define PACKED
+#endif
```

Comment:
Is this block really needed? I think we can safely assume we are always using GCC.

Reply:
Thank you @fauxpark! Probably a safe assumption 👍 

I've removed the conditional here.